src/avfoundation.cs

```diff
@@ -6713,8 +6754,8 @@ interface AVMetadataItemFilter {
 		AVMetadataItemFilter ForSharing { get; }
 	}
 
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
 	[NoWatch]
-	[NoTV]
```

Comment:
I guess that we need a TV with the correct version. Right now you went to the min version.

Reply:
I think the correct version is the min version(?) (& same on all the other places you commented same)
```cs
-API_AVAILABLE(macos(10.10), ios(6.0)) API_UNAVAILABLE(tvos, watchos)
+API_AVAILABLE(macos(10.10), ios(6.0), macCatalyst(14.0), tvos(9.0)) API_UNAVAILABLE(watchos)
 @interface AVMetadataObject : NSObject
 ```

Reply:
introspection will run on older tvOS simulators (gonna be 11.x) so we'll see if that header update is correct (or not)